CMakeLists.txt

```diff
@@ -369,6 +369,8 @@ IF(USE_HOME_DIR)
 ENDIF(USE_HOME_DIR)
 
 add_subdirectory(src)
+add_executable(main src/main.cpp)
```

Comment:
You don't want to add a new executable like this; it needs to be made a dependency of the existing executables.

zlib is already a dependency of the binaries in TILES mode, so you can see how this is done (in `src/CMakeLists.txt`, not this file) and mimic that.  Will need something like
```cmake
target_include_directories(cataclysm-common PUBLIC ${ZLIB_INCLUDE_DIRS})
target_link_libraries(cataclysm-common ${ZLIB_LIBRARIES})
```

Reply:
Not realizing there is a `src/CMakeLists.txt` was my major problem until now, thank you!